recipe/meta.yaml

```
@@ -149,6 +149,34 @@ outputs:
       commands:
         - echo "Hello World!"
 
+  # Contains LLVM tools with a version suffix
+  - name: llvm-tools-{{ major_ver }}
+    script: install_llvm.sh   # [unix]
+    script: install_llvm.bat  # [win]
+    build:
+      activate_in_script: true
+    requirements:
+      build:
+        - {{ stdlib('c') }}
+        - {{ compiler('cxx') }}
+        - cmake
+        - ninja
+        - python >=3
+        - libcxx {{ cxx_compiler_version }}  # [osx]
+      host:
+        - libcxx {{ cxx_compiler_version }}  # [osx]
+        - {{ pin_subpackage("libllvm" + major_ver, exact=True) }}
+        - libxml2
+        - zlib
+        - zstd
+      run:
+        - {{ pin_subpackage("libllvm" + major_ver, exact=True) }}
+    test:
+      commands:
+        - $PREFIX/bin/llc-{{ major_ver }} -version                               # [not win]
+        - if not exist "%LIBRARY_BIN%"\\llc-{{ major_ver }}.exe exit 1           # [win]
+        - llc-{{ major_ver }} -version                                           # [win]
```

Any reason this couldn't just be
```suggestion
        - llc-{{ major_ver }} -version
```
`$PREFIX/bin` is on the PATH, so not sure what we gain by the separation between win/non-win.

---

Because if these files are not packaged due to some issue in the build script, then any `llc-18` in the path would make the test pass

---

Sure, I meant that the risk is low that we pick up a random `llc-18` in our path.

---

We may want to add some test that `llvm-config` (version & unversioned) is absent.